lib/capabilities.js

```diff
@@ -161,6 +161,21 @@ var capabilities = {
       }
     ]
   },
+  debugger: {
+    name: 'debugger',
+    checks: [
+      {
+        check: function(cb) {
+          var dbg = require('./debugger');
+          if (dbg) {
+            cb(true);
+          } else {
+            cb(false, 'The native strong-debugger addon is not installed.');
```

Comment:
`Strong-debugger not compiled.` would fit the pattern of heap snapshot and strong-agent